package.json

```diff
@@ -69,5 +78,10 @@
   },
   "ember": {
     "edition": "octane"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "run-p lint:js:check format:check test"
```

Comment:
Should we do `pre-push` instead of `pre-commit` here?

Reply:
Good idea.